src/main/proto/com/kodypay/grpc/ecom/v1/ecom.proto

```diff
@@ -39,6 +39,7 @@ message PaymentInitiationRequest {
   optional ExpirySettings expiry = 13; // Nested message for expiry settings
   optional CaptureOptions capture_options = 14; // Optional capture settings
   optional bool is_pay_by_bank_accepted = 15; // Whether the payer can pay via pay by bank (Kody store must have pay by bank configured)
+  repeated PaymentMethods accepts_only = 16; // An inclusion list of accepted payment methods. If empty, all payment methods are accepted.
```

Comment:
Don't forget that this `PaymentMethods` list may not include all of the methods, for example ApplePay and GooglePay